### DISCUSSION TITLE
cannot finish install

### DISCUSSION THREAD

**Discussion by rionix88** [IndigoDosSantos/stable-cascade-one-click-installer#36] [Q&A]
the installation finish with error

 File "C:\Users\Nj bsyr\AppData\Local\Temp\pip-build-env-pxepzu0r\overlay\Lib\site-packages\setuptools\build_meta.py", line 311, in run_setup
          exec(code, locals())
        File "<string>", line 61, in <module>
      AssertionError: Could not find cmake in PATH
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Installation completed. Execute 'run.bat' script to start generating
Press Enter to move 'install.bat' to the 'stable-cascade-one-click-installer' directory and conclude the installation.

**Comment by Creepybits** [comment]
Have you tried to install Cmake and set it to path?

It seems like the logical first step.

https://cmake.org/download/
